src/backend/executor/process.ts

```diff
@@ -130,6 +153,41 @@ export class ExecutorProcess {
         ].join(' ');
     }
 
+    public getLogCmdLine(): string | undefined {
+        if (!workspace.workspaceFolders?.length) {
+            return undefined;
+        }
+
+        const workspaceFolder = workspace.workspaceFolders[0].uri.fsPath;
+
+        const getConfigAndReplaceVariables = (category: string, name: string): string | undefined => {
+            const configValue = workspace.getConfiguration(category).get<string>(name);
+            return configValue
+                ?.replace(/\${workspaceRoot}/g, workspaceFolder)
+                .replace(/\${workspaceFolder}/g, workspaceFolder)
+                .replace(/\${cwd}/g, process.cwd())
+                .replace(/\${env\.([^}]+)}/g, (sub: string, envName: string) => process.env[envName] ?? '');
+        };
+
+        const ccPath = getConfigAndReplaceVariables('codechecker.executor', 'executablePath')
+            ?? 'CodeChecker';
+        const ccFolder = getConfigAndReplaceVariables('codechecker.backend', 'outputFolder')
+            ?? path.join(workspaceFolder, '.codechecker');
+        const ccArguments = getConfigAndReplaceVariables('codechecker.executor', 'arguments') ?? '';
```

Comment:
These arguments are options for the `CodeChecker analyze` command and not for the `CodeChecker log` command. For example I set this argument to `--ctu` but the CodeChecker log command doesn't have such an option and it will fail.

So please create a separate option for it.
We can name the two arguments in the following names:
- Analysis Arguments
- Log arguments

It would also be good to create a separate command to show the full CodeChecker log command similar what we did for the CodeChecker analyze command.

Reply:
Fixed by simply removing all non-essential arguments.

I don't think there's a need for Log arguments, since the user is able to edit the command-line before running it.

The command is only meant as a guideline after all, because the user has to fill in their own build system (if different from make) per project anyways.